x-pack/plugins/security_solution/cypress/tasks/create_runtime_field.ts

```diff
@@ -19,11 +19,7 @@ export const createField = (fieldName: string): Cypress.Chainable<JQuery<HTMLEle
 
 export const assertFieldDisplayed = (fieldName: string, view: 'alerts' | 'timeline' = 'timeline') =>
   view === 'alerts'
-    ? cy
-        .get(
-          `[data-test-subj="events-viewer-panel"] [data-test-subj="dataGridHeaderCell-${fieldName}"]`
-        )
-        .should('exist')
+    ? cy.get(`[data-test-subj="dataGridHeaderCell-${fieldName}"]`).should('exist')
```

Comment:
We should avoid having conditionals on the testing code and have the assertions directly on the test. Also to have the locators encapsulated on the screens folder.

Reply:
I agree here. I have corrected it.